tests/Configurator/AddLinesConfiguratorTest.php

```diff
@@ -67,6 +67,26 @@ public function testLinesAddedToTopOfFile()
             $actualContents);
     }
 
+    public function testConvertsOptionsWhenConfiguring()
```

Comment:
As we assert only `expendTargetDir`, what to you think to rename tests ?

```suggestion
    public function testExpandTargetDirWhenConfiguring()
```

Reply:
Sure, thanks for the suggestion!